ui/process-shim.js

```diff
@@ -0,0 +1,4 @@
+// process-shim.js
+export let process = {
+  cwd: () => '',
+}
```

Comment:
To fix the runtime crash that reports can't run `process.cwd()`.

Reply:
Maybe better to distinguish who is calling this and checkout whether this hack will break things.

Reply:
Test it again after merging the master code, it seems nowhere report `process.cwd()` error anymore, so I remove this injection first.